challenge-1/picture-of-the-day/src/services/index.js

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable no-alert */
+import axios from 'axios';
+
+export async function getPicture(date) {
+  const apiKey = process.env.REACT_APP_API_KEY;
+  const searchUrl = `https://api.nasa.gov/planetary/apod?api_key=${apiKey}&date=${date}`;
```

Comment:
This could be unnecessary to store the constant, you could pass it directly on line 9. What do you think?

Reply:
youre right, it is unnecessary, and i believe it was set up as you suggest in an earlier iteration, just thought it was easier to read